rollup.config.js

```diff
@@ -0,0 +1,24 @@
+import resolve from '@rollup/plugin-node-resolve'
+import commonJs from '@rollup/plugin-commonjs';
+import {terser} from 'rollup-plugin-terser';
+
+export default {
+	input: 'src/webloader.js',
+	output: [
+		{
+			file: './dist/webloader.js',
+			format: 'iife',
+			name: 'webloader'
+		},
+		{
+			file: './dist/webloader.min.js',
+			format: 'iife',
+			plugins: [terser()]
+		}
+	],
+	external: ['http', 'https'],
+	plugins: [
+		resolve({preferBuiltins: true}),
```

Comment:
I'm not sure what `preferBuiltins` means for a web context. Unless you found it to be necessary, I would remove it.
```suggestion
		resolve({}),
```